iib/web/models.py

```diff
@@ -1198,6 +1195,15 @@ def from_json(cls, kwargs, batch=None):
 
         request_kwargs['binary_image'] = Image.get_or_create(pull_specification=binary_image)
 
+        distribution_scope = request_kwargs.pop('distribution_scope', None)
```

Comment:
nit: This logic is in two places, we may want to refactor at some point to its own validation function?

Reply:
other validation code for distribution scope is in iib/workers to refactor that, share module will be needed which will then have more then just this validation code and I don't want to include that change in this pull review